<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>returned</class>
 <widget class="QDialog" name="returned">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #6892d5;
color: #f8fcfb;</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>250</y>
     <width>201</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>No of books returned:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>180</y>
     <width>205</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>No. Of Book Assigned:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>250</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#0091bf;
border-radius:10px;
background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #e8f5e9;
color:#170a16;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>UPDATE</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>250</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#0091bf;border-radius:10px;
background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#0091bf;
background-color: rgb(255, 255, 255);
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#0091bf;
background-color: rgb(255, 255, 255);
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>80</y>
     <width>158</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choose the book:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>130</y>
     <width>157</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Umpush</family>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choose the User:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Bookman</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #f8fcfb;</string>
   </property>
   <property name="text">
    <string>Return Books</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
